ui/src/app/kvm/views/KVMList/LxdTable/LxdTable.test.tsx

```diff
@@ -53,4 +53,117 @@ describe("LxdTable", () => {
+            type: PodType.LXD,
+          }),
+        ],
+      }),
+    });
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={[{ pathname: "/kvm", key: "testKey" }]}>
+          <LxdTable />
+        </MemoryRouter>
+      </Provider>
+    );
+    const getLxdAddress = (rowNumber: number) =>
+      wrapper
+        .find("tbody TableRow")
+        .at(rowNumber)
+        .find("[data-test='lxd-address']");
+
+    // Sorted ascending by address by default
+    expect(getLxdAddress(0).text()).toBe("0.0.0.0");
+    expect(getLxdAddress(1).text()).toBe("172.0.0.1");
+    expect(getLxdAddress(2).text()).toBe("192.168.1.1");
+
+    // Change to sort descending by address
+    wrapper.find("[data-test='address-header'] button").simulate("click");
+    expect(getLxdAddress(0).text()).toBe("192.168.1.1");
+    expect(getLxdAddress(1).text()).toBe("172.0.0.1");
+    expect(getLxdAddress(2).text()).toBe("0.0.0.0");
+
+    // Change to no sort
+    wrapper.find("[data-test='address-header'] button").simulate("click");
+    expect(getLxdAddress(0).text()).toBe("172.0.0.1");
+    expect(getLxdAddress(1).text()).toBe("0.0.0.0");
+    expect(getLxdAddress(2).text()).toBe("192.168.1.1");
+  });
+
+  it("can update the LXD project sort order", () => {
+    const state = rootStateFactory({
+      pod: podStateFactory({
+        items: [
+          podFactory({
+            name: "pod-2",
+            power_address: "172.0.0.1",
+            type: PodType.LXD,
+          }),
+          podFactory({
+            name: "pod-3",
+            power_address: "192.168.1.1",
+            type: PodType.LXD,
+          }),
+          podFactory({
+            name: "pod-4",
+            power_address: "192.168.1.1",
+            type: PodType.LXD,
+          }),
+          podFactory({
+            name: "pod-1",
+            power_address: "172.0.0.1",
+            type: PodType.LXD,
+          }),
+        ],
+      }),
+    });
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={[{ pathname: "/kvm", key: "testKey" }]}>
+          <LxdTable />
+        </MemoryRouter>
+      </Provider>
+    );
+    const getLxdName = (rowNumber: number) =>
+      wrapper
+        .find("tbody TableRow")
+        .at(rowNumber)
+        .find("[data-test='pod-name']");
+
+    // Sorted ascending by name be default
+    expect(getLxdName(0).text()).toBe("pod-1");
+    expect(getLxdName(1).text()).toBe("pod-2");
+    expect(getLxdName(2).text()).toBe("pod-3");
+    expect(getLxdName(3).text()).toBe("pod-4");
+
+    // Change to sort descending by name
+    wrapper.find("[data-test='name-header'] button").simulate("click");
+    expect(getLxdName(0).text()).toBe("pod-2");
+    expect(getLxdName(1).text()).toBe("pod-1");
+    expect(getLxdName(2).text()).toBe("pod-4");
+    expect(getLxdName(3).text()).toBe("pod-3");
```

Comment:
This doesn't look descending

Reply:
Haha yeah, I guess the comment is not very clear. There's two groups with two pods each, but the pods are only sorted per group not the groups themselves. So in group 1, pod 1 and pod 2 are descending (2, 1), then in group 2, pod 3 and pod 4 are descending (4, 3)